.gitignore

```diff
@@ -1,3 +1,5 @@
 .DS_Store
 node_modules/
-config/
```

Comment:
Rather than removing this folder, please just `git add -f config/sample.env` if that's what you were trying to accomplish here.